packages/aragon-wrapper/src/rpc/handlers/addressIdentity.js

```diff
@@ -0,0 +1,14 @@
+export default function (request, proxy, wrapper) {
+  const [operation, address, providerName] = request
```

Comment:
Based on the [request](https://github.com/aragon/aragon.js/blob/master/packages/aragon-messenger/src/jsonrpc.js#L4) the destruction should happen on the `request.params` object